confirmation/models.py

```diff
@@ -100,6 +100,16 @@ def get_link_validity_in_days(self):
         # type: () -> int
         return settings.EMAIL_CHANGE_CONFIRMATION_DAYS
 
+class RemoteServerRegistrationManager(ConfirmationManager):
+    def get_activation_url(self, key, host=None):
```

Comment:
This function is the same as in the regular `ConfirmationManager` class; can we just use that (instead subclassing it as RemoteServerRegistrationManager)?